domains/fed.json

```
@@ -0,0 +1,8 @@
+{
+    "owner": {
+        "username": nitro-devs"
```

```suggestion
        "username": "nitro-devs"
```
Missing quotation mark.